config/environments/development.rb

```diff
@@ -30,6 +30,7 @@
   end
 
   # Don't care if the mailer can't send.
+  config.action_mailer.default_url_options = {host: 'localhost', port: 3000}
```

Comment:
@sinpai let's not save space if it's not needed:
```ruby
config.action_mailer.default_url_options = {
  host: 'localhost',
  port: 3000
}
```

Reply:
@sinpai by the way: what do you think about moving `3000` to a ENV variable? you might use `3000` in sevaral places across the app more, so you'll need to change value only in one place

Reply:
👍 moved to DEFAULT_PORT variable